### PR TITLE
DarkMode UI

### PR DESCRIPTION
**Pull request by TomaszJarzabek15**
New color theme added for dark mode, minor UI changes to make the app more functional on different screen sizes, mainly pomodoro and settings fragments xmls affected. Please review.

### CODE REVIEW

File: app/src/main/java/com/example/pomodorify/Settings.java [hunk 1]
```diff
@@ -24,6 +24,9 @@
 import android.widget.Toast;
 import android.widget.ToggleButton;
 
+import androidx.appcompat.app.AppCompatActivity;
```
Comment (NorbertKoziana) [inline app/src/main/java/com/example/pomodorify/Settings.java]: This line is not needed.

File: app/src/main/java/com/example/pomodorify/Settings.java [hunk 2]
```diff
@@ -183,7 +186,16 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                 setDarkThemePreferences.setDarkThemePreferences(isChecked);
 
                 if(isChecked){
-                    //darkThemeOn();
+                    // setting theme to night mode
+                    AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES);
+                }
+
+                // if the above condition turns false
```
Comment (NorbertKoziana) [inline app/src/main/java/com/example/pomodorify/Settings.java]: Remove this comment (I think the code is readable enough that it doesn't need commenting) or atleast write it in one line and thus make the if - else structure more readable.

**Review by NorbertKoziana — CHANGES_REQUESTED** [review]
Great effort! Please make the necessary adjustments.

**Review by TomaszJarzabek15 — COMMENTED** [review]
Changed the mentioned parts in the review

**Pull request merged**